src/interpreter.rs

```diff
@@ -20,9 +20,11 @@ impl Execute for Expression {
 
                 let (lower, any) = p.lhs.get_transformations();
 
+                // if not in "any" mode, then we need to check all values. 
```

Comment:
It seems that we already had these code in another PR. What is the relation between them?

Reply:
This check is an important fix thus the current PR is based on that one

> It seems that we already had these code in another PR. What is the relation between them?

Reply:
We should rebase this PR again since that change has been merged onto `master` already to simplify the diff.

Reply:
> We should rebase this PR again since that change has been merged onto `master` already to simplify the diff.

Fixing conflicts

Reply:
Rebase done @dndx